### ISSUE TITLE
add in_transaction read only property AbstractFileSystem

### ISSUE THREAD

**Issue by yonil7** [intake/filesystem_spec#684]
This property can be used by clients to check whether the fs is currently in transaction or not (similar to the internal _intrans attribute)

**Comment by martindurant** [comment]
Yes, this would be a fine addition. `_intrans` is already available, though - the underscore to me means that it is not expected to be used by end-users.
